README.md

```diff
@@ -1,7 +1,33 @@
+# OpenShift Serverless Previews Documentation
 
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides tutorials and information about
+additional, advanced use cases and integrations that are possible using
+OpenShift Container Platform and OpenShift Serverless, but are for development
+purposes only and are not officially supported by Red Hat.  This repository
+contains the source files for the OpenShift Serverless Previews documentation
+found at https://openshift-knative.github.io.
 
-Build and deploy Serverless applications using an event-driven infrastructure on Red Hat® OpenShift®.
+Current projects that fall into this category include
+
+* OpenShift Serverless Functions [Developer Preview] https://openshift-knative.github.io/docs/docs/functions/about-functions.html
+
+
+## Building the Site
+
+If you would like to build this site locally, you will need to have a recent
+version of Node.js installed, as well as the site formatting tool,
+[antorra](https://docs.antora.org/antora/2.3/install/install-antora/).
+
+To build the site:
+
+```
+antora antora-playbook.yml
+```
+
+You will find the output in `build/site`.
```

Comment:
I haven't done this in a minute but I'll check tomorrow or something if this is accurate. You also need to copy the files over then to replace the old ones if you actually want to publish the changes.

Reply:
OK - I've added those instructions, and also added a `.gitignore` file so the build directory doesn't end up getting added to the git repo